S/Snowball/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "Snowball"
+version = v"2.0.0"
+
+# Collection of sources required to build SDL2
+sources = [
+    "https://snowballstem.org/dist/libstemmer_c.tgz" =>
+    "054e76f2a05478632f2185025bff0b98952a2b7aed7c4e0960d72ba565de5dfc",
+    "./bundled",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libstemmer_c/
+cp ../CMakeLists.txt .
+rm Makefile
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=/opt/$target/$target.cmake
+make -j${nproc}
+make install
+"""
```

Comment:
You need to install some kind of license here.

Reply:
Looks like the CMake file does that in the last line.

Reply:
There is the file `COPYING` in the top directory.  It's not automatically installed because of the presence of the CMake file.

Reply:
A COPYING file gets copied in the `make install` step due to the cmake rule. It goes into `share/licences/Snowball/` but the auditor is now able to find it somehow.

Reply:
>install_license COPYING

Is that a CMake command? doesnt seem to work? In any case, as I say above, the tar bundle does contain COPYING in `share/licences/Snowball/`

Reply:
You're too British :stuck_out_tongue:

Reply:
> Is that a CMake command?

No, sorry, `install_license` is a [BinaryBuilder utility](https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Utilities-in-the-build-environment-1) to install the license file to the right directory.

In case my above wasn't clear enough, the directory should be called `share/licenses`, not `share/licences` :sweat_smile:

Reply:
Duh! Thanks!